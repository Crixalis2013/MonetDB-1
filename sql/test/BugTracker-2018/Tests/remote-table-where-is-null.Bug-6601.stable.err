stderr of test 'remote-table-where-is-null.Bug-6601` in directory 'sql/test/BugTracker-2018` itself:


# 12:25:26 >  
# 12:25:26 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=34060" "--set" "mapi_usock=/var/tmp/mtest-30679/.s.monetdb.34060" "--set" "monet_prompt=" "--forcemito" "--dbpath=/home/niels/scratch/rc-old/Linux-x86_64/var/MonetDB/mTests_sql_test_BugTracker-2018" "--set" "embedded_c=true"
# 12:25:26 >  


# 12:25:28 >  
# 12:25:28 >  "Done."
# 12:25:28 >  

