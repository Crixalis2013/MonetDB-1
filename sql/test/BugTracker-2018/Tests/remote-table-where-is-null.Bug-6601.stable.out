stdout of test 'remote-table-where-is-null.Bug-6601` in directory 'sql/test/BugTracker-2018` itself:


# 17:55:38 >  
# 17:55:38 >  "/usr/local/opt/python@2/bin/python2.7" "remote-table-where-is-null.Bug-6601.py" "remote-table-where-is-null.Bug-6601"
# 17:55:38 >  


# 12:25:28 >  
# 12:25:28 >  "Done."
# 12:25:28 >  

